### PR TITLE
create moyo header

### PR DESCRIPTION
**Pull request by buffi4ka**
https://buffi4ka.github.io/layout_moyo-header/
https://buffi4ka.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,25 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="container">
+        <a href="index.html" class="logo"><img src="./images/logo.png" alt="logo"></a>
```
Comment (33Orange) [inline src/index.html:16]: Let's use linebreaks in similar cases

```suggestion
        <a href="index.html" class="logo">
          <img src="./images/logo.png" alt="logo"/>
        </a>
```

**Review by 33Orange — CHANGES_REQUESTED** [review]
Your links don't work.

image

You have only 1 branch. You should change ONLY the branch `develop` and add your changes to the branch `gh-pages` via the terminal deploys command.

Please read this one more time and reproduce the steps:
https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github

image

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+  height: 60px;
+  text-transform: uppercase;
+  margin: auto;
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 1100px;
```
Comment (StasSokolov) [inline src/style.css:36]: container width should be `100%` as in the design

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+  color: var(--main-color);
+  font-size: var(--font-size);
+  line-height: var(--line-height);
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  background-color: rgb(241, 231, 231);
+}
+
+.header {
+  max-width: 1200px;
```
Comment (StasSokolov) [inline src/style.css:26]: there is no reason to set `max-width`, header without this property looks good

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,92 @@
...
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  color: var(--main-color);
+  font-size: var(--font-size);
+  line-height: var(--line-height);
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
```
Comment (StasSokolov) [inline src/style.css:21]: 🔥

File: src/index.html [hunk 5]
```diff
@@ -6,9 +6,29 @@
...
+        <a href="index.html" class="logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <a href="#" class="is-active link">Apple</a>
+          <a href="#" class="link">Samsung</a>
+          <a href="#" class="link">Smartphones</a>
+          <a href="#" class="link"
+            data-qa="hover">Laptops & Computers
+          </a>
```
Comment (StasSokolov) [inline src/index.html:25]: Снимок экрана 2023-09-06 в 10 27 28
Check in all places please

File: src/index.html [hunk 6]
```diff
@@ -6,9 +6,29 @@
...
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="container">
+        <a href="index.html" class="logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <a href="#" class="is-active link">Apple</a>
```
Comment (StasSokolov) [inline src/index.html:20]: Снимок экрана 2023-09-06 в 10 46 53
check this requirements please

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,92 @@
...
+  background-color: rgb(241, 231, 231);
+}
+
+.header {
+  max-width: 1200px;
+  height: 60px;
```
Comment (StasSokolov) [inline src/style.css:27]: Снимок экрана 2023-09-06 в 10 48 09
Check please

**Review by StasSokolov — CHANGES_REQUESTED** [review]
Good job! Please check comments and add `box-shadow` for header
Снимок экрана 2023-09-06 в 10 45 39

**Review by superpooperxxx — CHANGES_REQUESTED** [review]
Hey, I see that 6/8 tests fail. I highly recommend you attending QnA session to unblock you. Without all tests completed we cannot approve you task

**Review by superpooperxxx — CHANGES_REQUESTED** [review]
Hey, I see that you created Pull Request from your `master` branch. This is a mistake.
image

Come to QnA today, we will explain everything in greater detail

**Pull request closed without merging**